sdks/python/src/opik/integrations/llama_index/callback.py

```diff
@@ -160,9 +165,24 @@ def on_event_end(
                 llm_usage_info = event_parsing_utils.get_usage_data(payload)
                 span_data.update(**llm_usage_info.__dict__)
 
-                span_data.update(output=span_output).init_end_time()
+                span_data.update(
+                    output=span_output, error_info=error_info
+                ).init_end_time()
                 self._opik_client.span(**span_data.__dict__)
 
+                del self._map_event_id_to_span_data[event_id]
+            # # Orphaned events where the output was received after end_trace
```

Comment:
do we need this comment or it can be deleted?

Reply:
I've deleted it